src/components/Cards/item.tsx

```diff
@@ -7,20 +7,20 @@ export interface IItem {
   cardIcon?: EIconProps;
   label?: string;
   title: string;
-  text: string;
-  link?: {
+  text?: string;
+  links?: Array<{
```

Comment:
As this change represents a major update, the commit should populate the footer of the commit as [described](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit-message-footer) in the documentation